elements/campaign-display/fields/campaign-request-field.js

```diff
@@ -19,7 +19,7 @@ const CampaignRequestField = {
 
     fetchCampaignSuccess(state, response, store) {
       state.requestInFlight = false;
-      store.actions.handleFetchComplete(response);
+      setImmediate(() => store.actions.handleFetchComplete(response));
```

Comment:
I'd like to marinate that a little bit. We want a clean and clear way of describing actions and I'm not convinced actions calling actions, nested or otherwise, is a good idea.